### DISCUSSION TITLE
What happens to "orphaned" file attachments?

### DISCUSSION THREAD

**Discussion by Alhadis** [community/community#41225] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've always wondered what happens to files that're attached to a comment that's never posted. Do they linger forever in CDN purgatory, never to be seen again? Or are they pruned after a while; i.e., by a daemon to purge files with 0 views or references by comments and files after <var>𝑁</var> days?

I uploaded [this](https://user-images.githubusercontent.com/2346707/206707029-054f5d39-f3b1-43eb-9f8b-9975944d138c.png) last night whilst composing a comment I ended up not posting. A cursory skim of the REST API docs revealed no clear mechanism for deleting an unwanted file attachment (if such a thing is possible at all), so I'm curious as to how GitHub handles "junk" uploads that no doubt accumulate over time.

**Comment by MasterInQuestion** [comment]
I use the attachment feature no more.
    (migrated to dedicated ["attach"](https://github.com/MasterInQuestion/attach) repository)

    However to my experience, orphan attachments alike would be auto-deleted in less than 1 hour: (maybe even during editing the post...)
    If no explicit reference exists on GitHub.
